content/posts/apisix-wasm-support.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "How Does Apache APISIX Support WebAssembly?"
+date: 2023-11-14T09:46:06+05:30
+draft: true
+ShowToc: false
+ShowRelatedContent: false
+summary: "A look under the hood to see how Apache APISIX supports WebAssembly plugins."
+tags: ["apache apisix", "api gateway", "wasm"]
+categories: ["API Gateway"]
+cover:
+  image: "/images/apisix-wasm-support/binary-banner.jpeg"
+  alt: "Zeroes and ones."
+  caption: "It's all zeroes and ones if you look deep enough."
+  relative: false
+---
+
+[WebAssembly (Wasm)](https://webassembly.org/) has found its way outside the browser and has been widely adopted on the server side as in browsers with the development of [WebAssembly System Interface (WASI)](https://github.com/WebAssembly/WASI).
+
+In previous articles, we looked at how you can [build plugins for Apache APISIX in languages like Go using Wasm](/posts/tiny-apisix-plugin/) and how you can [optimize its performance](/posts/tinier-apisix-plugin/). We made it pretty clear that Wasm is here to stay in APISIX.
+
```

Comment:
Maybe you summarize and mention all the topics that are being covered in the article here. For example, _this article walks you through how the proxy Wasm spec is implemented, what is Wasm Module for Nginx, which programming models APISIX uses and how Wasm binaries are executed._

Reply:
In the conclusion?

Reply:
No, it is preferrable at the beginning of article as an agenda to highlight each section.  You mentioned this:

_But how does APISIX support these Wasm plugins? Do they have limitations? What will they look like in the future? In this "under the hood of APISIX" article, we will attempt to answer these questions and more._

However, you wrote only about limitations and future vision only in the summary.